src/theme/foundations/colors.ts

```diff
@@ -0,0 +1,82 @@
+import { Colors } from "@chakra-ui/react";
+
+// Colors taken from "../../styles/01-colors/*.scss"
```

Comment:
With these values being hardcoded, this adds another layer that we need ot keep an eye on if/when color values are updated.  We should consider using a method to export the SASS vars from the files in `01-colors` into a javascript object.  Additionally, the color SCSS files use a method to convert the SASS vars to the standard CSS vars.  That same functionality should be applied to all of the style files.  These, of course, are tasks for additional tickets.